command/core.py

```diff
@@ -0,0 +1,23 @@
+"""Calls the appropriate handler depending on the event data."""
+from command.commands.user import UserCommand
+
+
+class Core:
+    """Encapsulate methods for handling events."""
+
+    def __init__(self):
```

Comment:
I think eventually we'll want the command handler dictionary to be injected (or, if it's constructed locally, to have some way to add things to it from outside the class, such that we can add the commands from outside). I wouldn't bother changing it for this pull request, but keep it in mind for future ones.